layouts/funding_calls/list.html

```diff
@@ -40,154 +37,68 @@ <h4>Potential funding sources</h4>
-                  </div>
-                </div>`;
-
-            blockHTML += `
+  // Takes a URL and fetchs all open funding calls with label "Data Centre"
+  async function getDCFundingCalls(url, label="Data Centre") {
+    try {
+      // Fetch DC funding directly from Anubis using API
+      const response = await fetch(url);
+      const allData = await response.json();
+      const allFund = allData.calls.filter((f) => f.labels.includes(label))
+      return allFund.sort((a,b) => (a.closes > b.closes) ? 1 : (a.closes < b.closes) ? -1 : 0);
+    } catch (error) {
+      console.error('Error fetching events:', error);
+      return []; // Return empty array if API fails
+    }
+  };
+
+  // Code for funding sources starts here
+  function getFundingsCallsHTML(dcFundings) {   
+    let blockHTML = ""; 
+    // If no funds to show, display appropriate message
+    if(dcFundings.length !== 0) {
+      blockHTML += `<div class="row g-2 mt-3 mb-4">`;
+      dcFundings.forEach((item) => {
+        const close_date = new Date(item.closes);
+        const close_date_formatted = close_date.toLocaleString('en-SE', { timeZone: 'Europe/Stockholm', 'month': 'short', day:'numeric', year: 'numeric', hourCycle:'h12', hour: 'numeric', minute: 'numeric' });
+        blockHTML += `
+        <div class="col-md-6 col-lg-4">
+          <div class="card-box">
+            <a href="${item.href}" target="_blank">
+              <h5>${item.title}</h5>
+              <div>
+                <b>Deadline:</b> ${close_date_formatted + ' (Swedish time)'}
               </div>
-            </div>`;
-          });
-          
-        blockHTML += `</div>`;
-      } else {
-        var blockHTML = `<div class="alert alert-secondary mt-4">
-                            There are currently no open funding calls from SciLifeLab related to data-driven life science.
-                        </div>`
-      }
-      
-      var dynamicSection = document.getElementById('fundings-calls-section');
-      dynamicSection.innerHTML = blockHTML;
-    };
-
-
-    // Call the function to fetch, parse and display jobs
-    getDataFromUrl(
-      decodeURIComponent("https://blobserver.dc.scilifelab.se/blob/data_platform_funding_calls.json"),
-      addFundingsCalls
-    );
-
-  }); // Document "DOMContentLoaded" trigger ends here
+            </a>
+          </div>
+        </div>`;
+      }); // for each loop ends here
+      blockHTML += `</div>`;
+    } else {
+      blockHTML += `<div class="alert alert-secondary mt-4">
+                          No funding or calls are currently available from SciLifeLab Data Centre.
+                      </div>`
+    }
+    return blockHTML;
+  };
+
+  // Functions need to be run when the DOM is ready
+  document.addEventListener("DOMContentLoaded", async () => {
+    let fundingHTML = "";
+    try {
+      const dcCalls = await getDCFundingCalls("https://anubis.scilifelab.se/api/calls/open");
+      fundingHTML += getFundingsCallsHTML(fundings.calls);
```

Comment:
Uses `fundings.calls`, but the variable is `dcCalls`. This will throw a `ReferenceError`.
```suggestion
      fundingHTML += getFundingsCallsHTML(dcCalls);
```

Reply:
Good catch, I forgot to revert after testing locally